software/openvisualizer/openvisualizer/eventBus/eventBusClient.py

```diff
@@ -147,10 +147,10 @@ def _signalsEquivalent(self,s1,s2):
         elif type(s1)==type(s2)==tuple:
             assert len(s1)==len(s2)==3
             for i in range(3):
-                if (s1[i]==s2[i]) or (s1[i]==self.WILDCARD) or (s2[i]==self.WILDCARD):
-                    return True
-                else:
+                if (s1[i]!=s2[i]) and (s1[i]!=self.WILDCARD) and (s2[i]!=self.WILDCARD):
                     return False
+            else:
```

Comment:
Hi @kb2ma , Thanks a lot for catching this! It seems the two lines code are not aligned well. Could you fix it?  Thanks!